src/index.ts

```diff
@@ -60,9 +59,19 @@ export default new Vue({
       await dashboardStore.setEnv(production)
     } else {
       console.log("multiple envs")
-      dashboardStore.setEnvs([production, extDev, stage, dev])
-      // default
-      await dashboardStore.setEnv(stage)
+      const userSelected = localStorage.getItem("selectedConfig")
+      const multipleEnvs = [production, extDev, dev]
+
+      dashboardStore.setEnvs(multipleEnvs)
+
+      if (userSelected) {
+        // if user has already selected a config before
+        const selectedEnv = multipleEnvs.find((env) => env.name === userSelected)!
+        await dashboardStore.setEnv(selectedEnv)
+      } else {
+        // default
+        await dashboardStore.setEnv(extDev)
```

Comment:
Why does this default to Extdev? Shouldn't it default to Basechain?

Reply:
It default to ext-dev when not on production.